pkg/aci/aci.go

```diff
@@ -33,11 +33,17 @@ import (
 )
 
 var (
+	// ErrChmod - Error message for error changing file permission
```

Comment:
Unless golint is specifically asking for it, you could just remove the error name - and start at Error message.